### PR TITLE
feat: create OrderResolver

### PR DESCRIPTION
**Pull request by PoorlyDefinedBehaviour**

### CODE REVIEW

File: server/src/Resolvers/Order.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+@Resolver(() => Order)
+export default class OrderResolver {
+  @Mutation(() => Order)
+  @UseMiddleware(Authenticated)
+  async createOrder(
+    @Arg("products", () => [Int]) products: number[],
+    @Ctx() { auth }
+  ) {
+    console.log("auth.user.id", auth.user)
```
Comment (JVictorV) [inline server/src/Resolvers/Order.ts]: remover console.log

File: server/src/Resolvers/Order.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    const order = Order.create({ user: auth.user })
+
+    order.products = await Product.findByIds(products)
+
+    console.log("ordeR", order)
```
Comment (JVictorV) [inline server/src/Resolvers/Order.ts]: remover console.log

**Review by JVictorV — CHANGES_REQUESTED** [review]
Ficaram alguns console.logs no OrderResolver

**Comment by PoorlyDefinedBehaviour** [comment]
> Ficaram alguns console.logs no OrderResolver

Tirei os `console.log` @JVictorV

**Pull request merged**